src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java

```diff
@@ -0,0 +1,84 @@
+package com.google.checkstyle.test.chapter5naming.rule53camelcase;
+
+class InputCamelCaseDefined {
+
+    int newCustomerId;
+
+    String innerStopwatch;
+
+    boolean supportsIpv6OnIos;
+
+    void XmlHttpRequest() {}
+
+    void YouTubeImporter() {}
+
+    void YoutubeImporter() {}
+
+    class InnerGood {
+
+        int newCustomerId;
+
+        String innerStopwatch;
+
+        boolean supportsIpv6OnIos;
+
+        void XmlHttpRequest() {}
+
+        void YouTubeImporter() {}
+
+        void YoutubeImporter() {}
+    }
+
+        InputCamelCaseDefined anonymousGood
+            = new InputCamelCaseDefined() {
+
+            int newCustomerId;
+
+            String innerStopwatch;
+
+            boolean supportsIpv6OnIos;
+
+            void XmlHttpRequest() {}
+
+            void YouTubeImporter() {}
+
+            void YoutubeImporter() {}
+    };
+}
+
+class AbbreviationsIncorrect {
+
+    int newCustomerID;
+    // violation above '.* 'newCustomerID' .* no more than '1' .* capital letters.'
```

Comment:
// violation above ' 'newCustomerID' .* more than '1' .* capital letters.'

Reply:
done

Reply:
@Zopsss , please remove heading `.*`

Reply:
And single quotes also can be removed

Reply:
Ok I see that you tried , but there is problems

Reply:
But we can remove single quotes

Reply:
fixed by https://github.com/checkstyle/checkstyle/issues/15138
commit in this PR.